Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Best practices from: https://nodejs.org/en/docs/guides/nodejs-docker-webapp/
+FROM node:carbon
+
+# BEGIN root
+USER root
+RUN yarn add global grunt-cli \
+    && yarn global add loopback-sdk-angular-cli
+
+RUN mkdir -p /usr/src/app
+
+COPY . /usr/src/app
+RUN chown -R node:node /usr/src/app
```

Comment:
Does the app need the ability to modify it's files during run time? I mean something like caches, some bytecode stored at FS or anything else.
Will it work if `npm start` is run as another `USER`?

Reply:
You don't need it at runtime, but you do need permissions to do `npm install` and `npm build`